src/Tools/dotnet-counters/CounterMonitor.cs

```diff
@@ -104,6 +104,32 @@ public async Task<int> Monitor(CancellationToken ct, List<string> counter_list,
             }
         }
 
+        // Use EventPipe CollectTracing2 command to start monitoring. This may throw.
+        private void RequestTracingV2(string providerString)
+        {
+            var configuration = new SessionConfigurationV2(
+                                        circularBufferSizeMB: 1000,
```

Comment:
I just realized that the `circularBufferSizeMB` is hardcoded here, and it's not user input.